### DISCUSSION TITLE
Organization

### DISCUSSION THREAD

**Discussion by LukeKolin** [community/community#33374] [New to GitHub]
A few months back I was experimenting with organizations in GitHub for code for my hobby club, and successfully created an organization. My user account isn't part of the organization anymore and I can't seem to log into the org to add myself back to it.

Is my org orphaned forever?

Cheers

**Comment by queenofcorgis** [comment]
Hi there! I have a few thoughts 💭 

* When you click the dropdown from your profile icon in the upper righthand corner and navigate to "My Organizations" is it listed under there? 
* Is anyone else in the organization that can link you back into it?
* If the answer is no to either of the questions above, I'd reach out to our lovely [Support](https://support.github.com/) folks as they'll be able to best assist with an account-specific situation like this one. 

Good luck and keep us updated!
